src/DependenciesPackager/Program.cs

```diff
@@ -174,6 +178,11 @@ static int Main(string[] args)
                 "The prefix to use for the zip file name",
                 CommandOptionType.SingleValue);
 
+            var uploadToAzure = app.Option(
+                "--upload-to-azure",
```

Comment:
Seems kinda out of place with the rest of the stuff this thing does. Would it make sense to have a different task that does the upload?